pkg/converter/convert_unix.go

```diff
@@ -130,134 +130,76 @@ func unpackNydusTar(bootDst, blobDst string, ra content.ReaderAt) error {
 	}
 	defer blob.Close()
 
-	if err = unpackBlobFromNydusTar(ra, blob); err != nil {
+	if err = unpackFileFromNydusTar(ra, blobNameInTar, blob); err != nil {
```

Comment:
Can we rename NydusTar to NydusArchive which archives both blob and bootstrap borrowing the tar header structure describing them

Reply:
Fixed.